src/Synapse/Validator/Constraints/I18n/BelongsToEntity.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace Synapse\Validator\Constraints\I18n;
+
+use Synapse\Validator\Constraints\BelongsToEntity as ParentConstraint;
+
+class BelongsToEntity extends ParentConstraint
+{
+    public $message = 'DOES_NOT_BELONG_TO_ENTITY';
```

Comment:
We've gotten by all this time with the message templates being public instance properties. Why change now?